heisenbridge/network_room.py

```diff
@@ -132,6 +133,7 @@ def init(self):
         self.pills_length = 2
         self.pills_ignore = []
         self.autoquery = True
+        self.color = True
```

Comment:
I wonder if we should plan to have Matrix->IRC color support as well and rename this to something like `irc_colors` _or_ store it like an enum `[None, "irc", "matrix", "both"]` (this is a pattern that's already used) as it becomes ambiguous when expanded.

Reply:
I don't plan to implement Matrix->IRC color support, so I leave the decision of how to store the variable to you.